includes/rules/missing_headings.php

```diff
@@ -13,12 +13,17 @@
  * @return array
  */
 function edac_rule_missing_headings( $content, $post ) {
+	if ( ! EDAC_KEY_VALID ) {
```

Comment:
This rule should run on the free plugin. It may have been confusing on the ticket where it said "set this up so that it can be a future setting in Pro". I believe what was meant by that was that the word count of 400 could be set in the Pro plugin settings. But that is outside the scope of this issue. The EDAC_KEY_VALID check can be removed.